File: Client/QRCode/src/com/entity/Task.java
package com.entity;

import java.io.Serializable;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

@DatabaseTable(tableName = "PatrolTasks")
public class Task implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	@DatabaseField(generatedId = true)
	int id;
	@DatabaseField
	String TaskID;
	@DatabaseField
	String TaskType;
	@DatabaseField
	String UserName;
	@DatabaseField
	String Description;
	@DatabaseField
	String TaskDetail;
	@DatabaseField
	String TaskStatus;
	@DatabaseField
	String Time;
	
	public Task()
	{
		
	}
	
	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getTaskStatus() {
		return TaskStatus;
	}

	public void setTaskStatus(String taskStatus) {
		TaskStatus = taskStatus;
	}
	public String getTaskID() {
		return TaskID;
	}
	public void setTaskID(String taskID) {
		TaskID = taskID;
	}
	public String getTaskType() {
		return TaskType;
	}
	public void setTaskType(String taskType) {
		TaskType = taskType;
	}
	public String getUserName() {
		return UserName;
	}
	public void setUserName(String userName) {
		UserName = userName;
	}
	public String getDescription() {
		return Description;
	}
	public void setDescription(String description) {
		Description = description;
	}

	public String getTaskDetail() {
		return TaskDetail;
	}

	public void setTaskDetail(String taskDetail) {
		TaskDetail = taskDetail;
	}

	public String getTime() {
		return Time;
	}
	public void setTime(String time) {
		Time = time;
	}
	
	
}

File: Server/QRWebService/QRWebService/Model/Task.cs
using System;
namespace QR.Model
{
	/// <summary>
	/// 1
	/// </summary>
	[Serializable]
	public partial class Task
	{
		public Task()
		{}
		#region Model
		private int _taskid;
		private string _tasktype;
		private string _username;
		private string _description;
		private DateTime? _time;
		private string _taskstatus;
		private string _taskdetail;
		/// <summary>
		/// 
		/// </summary>
		public int TaskID
		{
			set{ _taskid=value;}
			get{return _taskid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string TaskType
		{
			set{ _tasktype=value;}
			get{return _tasktype;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string UserName
		{
			set{ _username=value;}
			get{return _username;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string Description
		{
			set{ _description=value;}
			get{return _description;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime? Time
		{
			set{ _time=value;}
			get{return _time;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string TaskStatus
		{
			set{ _taskstatus=value;}
			get{return _taskstatus;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string TaskDetail
		{
			set{ _taskdetail=value;}
			get{return _taskdetail;}
		}
		#endregion Model

	}
}

